adhocracy-plus/config/settings/base.py

```diff
@@ -544,3 +544,4 @@
 
 AI_API_AUTH_TOKEN = ''
 AI_API_URL = 'https://kosmo-api-dev.liqd.net/api/classify/'
+AI_USAGE = False
```

Comment:
I think, instead of adding this to pillar, we could set it to True here. Then if we deploy and forget to add the token to the pillar, we will get a warning!

Reply:
Right!